### ISSUE TITLE
Missing outlink icon for WCAG for hyperlinks

### ISSUE THREAD

**Issue by JessicaFrank** [CCALI/a2jviewer#40]
![image](https://user-images.githubusercontent.com/7662028/89826631-284e2a80-db1c-11ea-9863-60b7bfa4d0d6.png)

Chrome browser
